### PR TITLE
Fixes #316 - Addition of disable_windows_task matcher

### LINKED ISSUE #316: Add `disable_windows_task` ChefSpec matcher
Currently there is no matcher for the "disable" action on the `window_task` resource.

### PR DESCRIPTION
**Pull request by jrnt30**

### CODE REVIEW

File: libraries/matchers.rb [hunk 1]
```diff
@@ -181,6 +181,31 @@ def create_windows_task(resource_name)
...
+  # +windows_task+:
+  #
+  #     windows_task 'mytask' do
+  #       action :disable
+  #     end
+  #
+  # The Examples section demonstrates the different ways to test a
+  # +windows_task+ resource with ChefSpec.
+  #
+  # @example Assert that a +windows_task+ was created
```
Comment (mwrock) [inline libraries/matchers.rb]: should be "was disabled"

Comment (jrnt30) [inline libraries/matchers.rb]: Corrected, thanks

**Comment by mwrock** [comment]
:+1: 
cc @chef-cookbooks/client-windows

**Comment by smurawski** [comment]
:+1:

**Pull request merged**